common/include/pcl/pcl_macros.h

```diff
@@ -301,18 +301,6 @@ pcl_round (float number)
 #define FIXED(s) \
   std::fixed << s << std::resetiosflags(std::ios_base::fixed)
 
-#ifndef ERASE_STRUCT
```

Comment:
Note to maintainers: API break or deprecation, depending on author's future commits

Reply:
Restored, will reassess deprecation